pulp_rpm/app/viewsets.py

```diff
@@ -183,25 +317,125 @@ class RpmRepositoryVersionViewSet(RepositoryVersionViewSet):
 
     parent_viewset = RpmRepositoryViewSet
 
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_repository_model_or_obj_perms:rpm.view_rpmrepository",
+            },
+            {
+                "action": ["destroy"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_repository_model_or_obj_perms:rpm.delete_rpmrepository",
```

Comment:
maybe it's worth also adding separate permission for repo_version_deletion?https://github.com/pulp/pulp_container/blob/main/pulp_container/app/viewsets.py#L900
@dralley @mdellweg

Reply:
followed link and see that there is separate permission for repo_version_deletion but it is gained with repo creation and not repo_version_creation so it looks to me like a duplicated perm. What do I miss there @mdellweg?

Reply:
@ipanova Do you kow of a particular strong use case for this that applies to the RPM plugin?